spec/lib/repo/git_revision_spec.rb

```diff
@@ -2,6 +2,25 @@
 
 describe Repository::GitRevision do
   context 'with a git repo' do
+    before(:context) do
+      
+      # Make sure gitolite-admin repo is cloned in test environment
+      ga_repo = Gitolite::GitoliteAdmin.new(
+        "#{::Rails.root}/data/test/repos/gitolite-admin", GITOLITE_SETTINGS)
+
+      # Bring the repo up to date
+      ga_repo.reload!
+
+      # Grab the gitolite admin repo config
+      conf = ga_repo.config
+
+      # Make sure repo exists, if not make it
+      repo = ga_repo.config.get_repo('test_repo_workdir')
+      if repo.nil?
+         Repository::GitRepository.create("test_repo_workdir")
```

Comment:
Use 2 (not 3) spaces for indentation.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.